deploy/backplane/osd-17545/30-OSD-17545.CronJob.yaml

```diff
@@ -0,0 +1,60 @@
+  namespace: openshift-sre-pruning
+  annotations:
+    kubernetes.io/description: Mitigate https://issues.redhat.com/browse/OSD-17545
+spec:
+  failedJobsHistoryLimit: 3
+  successfulJobsHistoryLimit: 3
+  schedule: "*/30 * * * *" # Every thirty minutes
+  concurrencyPolicy: Replace
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      ttlSecondsAfterFinished: 3600
+      template:
+        metadata:
+          name: osd-17545
+          namespace: openshift-sre-pruning
+          annotations:
+            kubernetes.io/description: Mitigate https://issues.redhat.com/browse/OSD-17545
+        spec:
+          affinity:
+            nodeAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+              - preference:
+                  matchExpressions:
+                  - key: node-role.kubernetes.io/infra
+                    operator: Exists
+                weight: 1
+          tolerations:
+            - effect: NoSchedule
+              key: node-role.kubernetes.io/infra
+              operator: Exists
+          containers:
+          - name: osd-17545
+            image: image-registry.openshift-image-registry.svc:5000/openshift/tools:latest
+            imagePullPolicy: Always
+            securityContext:
+              allowPrivilegeEscalation: false
+              capabilities:
+                drop:
+                - ALL
+              runAsNonRoot: true
+            command:
+            - /bin/bash
+            args:
+            - -c
+            - |
+              # cleanup rolebindings
+              oc get rolebindings -A | grep -e backplane-lpsre-monitoring-system:serviceaccounts:openshift-backplane-lpsre -e backplane-ocs-consumer-cr-admins-system:serviceaccounts:openshift-backplane-lpsre -e backplane-lpsre-monitoring-system:serviceaccounts:openshift-backplane-lpsre -e backplane-lpsre-monitoring-system:serviceaccounts:openshift-backplane-lpsre -e backplane-ocs-provider-cr-admins-system:serviceaccounts:openshift-backplane-lpsre -e backplane-lpsre-addon-operator-olm-admin-system:serviceaccounts:openshift-backplane-lpsre -e backplane-mcg-osd-cr-admins-system:serviceaccounts:openshift-backplane-mcg -e backplane-mtsre-monitoring-system:serviceaccounts:openshift-backplane-mtsre -e backplane-ocs-consumer-cr-admins-system:serviceaccounts:openshift-backplane-mtsre -e backplane-ocs-provider-cr-admins-system:serviceaccounts:openshift-backplane-mtsre -e backplane-openshift-connectors-cr-admins-system:serviceaccounts:openshift-backplane-mtsre -e backplane-mtsre-addon-operator-olm-admin-system:serviceaccounts:openshift-backplane-mtsre -e backplane-ocs-consumer-cr-admins-system:serviceaccounts:openshift-backplane-odf -e backplane-ocs-provider-cr-admins-system:serviceaccounts:openshift-backplane-odf -e backplane-lpsre-monitoring-system:serviceaccounts:openshift-backplane-odf-sre -e backplane-srep-system:serviceaccounts:openshift-backplane-srep | awk '{print "oc -n " $1 " delete rolebinding " $2}' | sh
```

Comment:
Is this idempotent? It doesn't look like it using grep/awk/sh

Reply:
Yea, if nothing is matched with the grep then there's no input to awk therefore nothing to sh.  I tested multiple runs in a row to confirm.